package.json

```diff
@@ -53,11 +53,5 @@
     "debounce": "^1.1.0"
   },
   "simple-git-hooks": {
-    "pre-commit": "npx lint-staged"
-  },
-  "lint-staged": {
```

Comment:
> Add nano-staged and simple-git-hooks to lint files on commit and run tests on push

Why have you removed it?

Reply:
because in a next PR will push nano-staged